### PR TITLE
[make_arm_controller_model] Account for attached mass

### PR DESCRIPTION
**Pull request by EricCousineau-TRI**
Follow-up to #20513 
Towards Anzu PR 11156

- - -
This change is [Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)

### CODE REVIEW

**Review by EricCousineau-TRI — COMMENTED** [review]
+@zachfang Could I ask for your review on this PR?

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-:-NnH57Bb8VZvI3-079lh:b-quygpl)* status: LGTM missing from assignee zachfang, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @EricCousineau-TRI)

**Comment by zachfang** [comment]
Sure thing. From a quick check, CI is complaining about a missing namespace.
```
[9:37:09 AM]  manipulation/util/make_arm_controller_model.cc:179:11: error: no template named 'Body'; did you mean 'multibody::Body'?
```

**Comment by RussTedrake** [comment]
Just fwiw... i feel that this entire workflow should go away, and have been working on a PR train to enable "ReplaceJointWithWeld".  (#20711 and #20720 are the first two PRs in the train).  I think the better way to make the controller plant is to load the subset of the model, including the gripper (and cameras, etc) and then replace all non-weld joints with weld joints to make everything downstream of the arm into a lumped inertia... (but one that still has all of the kinematics/frames, etc of the original model).

**Comment by EricCousineau-TRI** [comment]
Most definitely (I would love to remove this bespoke workflow), and yeah, that'd be awesome!

Addisu and I tried to squeeze in something similar via #17961: basically, a lasso tool + find replace on joints. However, we ultimately stalled due to other priorities.
With this, though, you do not need to reload anything; you select, then copy and paste onto a pre-finalized plant.
You can then decide how you want to change joints to welds - either via copy mappings (as illustrated in that PR), or via pre-finalize modifications as you stated.
Do you want me to queue that up in the next month or so? I would love to finish that thread out, if you think that approach is useful.

For now, I'd still like to land this small acute fix (assuming it's close to landing as-is) to unblock existing issues.

**Comment by EricCousineau-TRI** [comment]
> Sure thing. From a quick check, CI is complaining about a missing namespace.

Done, thanks!

**Comment by RussTedrake** [comment]
I haven't thought fully enough about your subgraphs to give a fully formed opinion, but cloning pieces of the plant into a new copy (without reloading) seems very useful.  I think pre-finalize mutability is also valuable.

**Review by EricCousineau-TRI — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)* status: 1 unresolved discussion, LGTM missing from assignee zachfang, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes

___
*[`manipulation/util/make_arm_controller_model.cc` line 87 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnL56W01BdpEgtKIgge:-NnL56W01BdpEgtKIggf:bbwn7hs) ([raw file](https://github.com/RobotLocomotion/drake/blob/e5b9b976a3ca521180d738b094d27ab37383de2c/manipulation/util/make_arm_controller_model.cc#L87)):*
> ```cpp
>       arm_X_WC);
> 
>   const auto sim_arm_bodies = simulation_plant.GetBodyIndices(arm_model_index);
> ```

Working: This is a bug; it should be `sim_arm_model_index`.

**Review by EricCousineau-TRI — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)* status: LGTM missing from assignee zachfang, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes

___
*[`manipulation/util/make_arm_controller_model.cc` line 87 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnL56W01BdpEgtKIgge:-NnL7Lqq5_4yepQFVVJ0:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/e5b9b976a3ca521180d738b094d27ab37383de2c/manipulation/util/make_arm_controller_model.cc#L87)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

Working: This is a bug; it should be `sim_arm_model_index`.
</blockquote></details>

Done.

**Review by zachfang — COMMENTED** [review]
:lgtm: feature.

> Reviewed 3 of 5 files at r1, 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-:-NnMJQME7bzhv8_WIHp9:b-n37teh)* status: 6 unresolved discussions, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @EricCousineau-TRI)

___
*[`manipulation/util/make_arm_controller_model.cc` line 87 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMEbKa-jMH7we0YnkR:-NnMEbKb8FtYr9HPN8aJ:bg0of0x) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/make_arm_controller_model.cc#L87)):*
> ```cpp
>       arm_X_WC);
> 
>   const auto sim_arm_bodies =
> ```

nit. To be explicit it's a vector of `BodyIndex` and is consistent with `sim_gripper_body_indices`.



_Suggestion:_
```cpp
sim_arm_body_indices
```
___
*[`manipulation/util/make_arm_controller_model.cc` line 89 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMF3RF-EwNghvZrNWu:-NnMF3RF-EwNghvZrNWv:be2sps8) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/make_arm_controller_model.cc#L89)):*
> ```cpp
>   const auto sim_arm_bodies =
>       simulation_plant.GetBodyIndices(sim_arm_model_index);
>   std::set<BodyIndex> sim_bodies_accounted_for;
> ```

BTW. Maybe we can initialize the set with a vector during construction time.



_Suggestion:_
```cpp
std::set<BodyIndex> sim_bodies_accounted_for(sim_arm_bodies.begin(), sim_arm_bodies.end());
```
___
*[`manipulation/util/test/iiwa7_wsg_cameras.dmd.yaml` line 1 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMHsgX81ErY4MxttGP:-NnMHsgY4UNem1ZDj5jH:b6g8mzm) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/iiwa7_wsg_cameras.dmd.yaml#L1)):*
> ```YAML
> directives:
> ```

nit. Can we add a simple comment to convey that the purpose of this dmd file is to test extra welded bodies on the arm (or something similar)?



_Code quote:_
```YAML
directives:
```
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 52 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMITId4JjPOu1BGEFe:-NnMITId4JjPOu1BGEFf:bi7cmsk) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/make_arm_controller_model_test.cc#L52)):*
> ```cpp
>             "drake/manipulation/models/wsg_50_description/sdf/"
>             "schunk_wsg_50.sdf")) {
>     // Add empty model intsance to ensure model instance lookups are correct
> ```

nit. Typo.



_Suggestion:_
```cpp
Add an `empty` model instance
```
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 164 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMIeoRAbQJI9uAK_QN:-NnMIeoRAbQJI9uAK_QO:bb7lc56) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/make_arm_controller_model_test.cc#L164)):*
> ```cpp
>   // world and one for a default model instance for unspecified modeling
>   // elements. Note that we add an additional model instance ("empty") to the
>   // the simulation plant to have meaningfully different model instance
> ```

nit.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 308 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMHTE17JzYRM5hsK-b:-NnMHTE17JzYRM5hsK-c:bb7lc56) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/make_arm_controller_model_test.cc#L308)):*
> ```cpp
>       "drake/manipulation/util/test/iiwa7_wsg_cameras.dmd.yaml"));
>   Parser parser{sim_plant_};
>   std::vector<ModelInstanceInfo> models_from_directives =
> ```

nit.



_Suggestion:_
```cpp
const std::vector<ModelInstanceInfo>
```

**Review by EricCousineau-TRI — COMMENTED** [review]
Thanks! +@SeanCurtis-TRI for platform review, please

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-:-NnM_xVpB6rWOLgdxodo:bqwb2jt)* status: LGTM missing from assignee SeanCurtis-TRI(platform), missing label for release notes (waiting on @EricCousineau-TRI)

___
*[`manipulation/util/make_arm_controller_model.cc` line 87 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMEbKa-jMH7we0YnkR:-NnM_arJ4wihTdoj5PAI:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/make_arm_controller_model.cc#L87)):*
<details><summary><i>Previously, zachfang wrote…</i></summary><blockquote>

nit. To be explicit it's a vector of `BodyIndex` and is consistent with `sim_gripper_body_indices`.
</blockquote></details>

Done.
___
*[`manipulation/util/make_arm_controller_model.cc` line 89 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMF3RF-EwNghvZrNWu:-NnM_bpbChb9QYcv3jnM:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/make_arm_controller_model.cc#L89)):*
<details><summary><i>Previously, zachfang wrote…</i></summary><blockquote>

BTW. Maybe we can initialize the set with a vector during construction time.
</blockquote></details>

Done.
___
*[`manipulation/util/test/iiwa7_wsg_cameras.dmd.yaml` line 1 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMHsgX81ErY4MxttGP:-NnM_czn2ET2h8BzUFXy:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/iiwa7_wsg_cameras.dmd.yaml#L1)):*
<details><summary><i>Previously, zachfang wrote…</i></summary><blockquote>

nit. Can we add a simple comment to convey that the purpose of this dmd file is to test extra welded bodies on the arm (or something similar)?
</blockquote></details>

Done.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 52 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMITId4JjPOu1BGEFe:-NnM_sH63lt7vhQGUDOW:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/make_arm_controller_model_test.cc#L52)):*
<details><summary><i>Previously, zachfang wrote…</i></summary><blockquote>

nit. Typo.
</blockquote></details>

Done.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 164 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMIeoRAbQJI9uAK_QN:-NnM_hDa1QI-Hj6l_le0:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/make_arm_controller_model_test.cc#L164)):*
<details><summary><i>Previously, zachfang wrote…</i></summary><blockquote>

nit.
</blockquote></details>

Done.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 308 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-NnMHTE17JzYRM5hsK-b:-NnM_ihZCSZuRVuQELXx:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/07c50166af4efff92fae011ce8246483ef2eb40b/manipulation/util/test/make_arm_controller_model_test.cc#L308)):*
<details><summary><i>Previously, zachfang wrote…</i></summary><blockquote>

nit.
</blockquote></details>

Done.

**Review by SeanCurtis-TRI — COMMENTED** [review]
:LGTM: With some small tweaks to clarify and keep things up to date.

> Reviewed 2 of 5 files at r1, 3 of 3 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-:-Nne7wBUDO3HjiDtOX1L:bfrhuxf)* status: 5 unresolved discussions, missing label for release notes (waiting on @EricCousineau-TRI)

___
*[`manipulation/util/make_arm_controller_model.cc` line 87 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nndz9En6bg01-57wN5x:-Nndz9EoEWNgoutMc3Ao:bm2pb39) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/make_arm_controller_model.cc#L87)):*
> ```cpp
>       arm_X_WC);
> 
>   const auto sim_arm_body_indices =
> ```

BTW Perhaps too aggressive a use of `auto` here? Particularly as it's not just a temporary used to instantiate `sim_bodies_accounted_for`, but is used in your new for loop?
___
*[`manipulation/util/make_arm_controller_model.cc` line 177 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne1tyY69VK4GyaKTes:-Nne1tyY69VK4GyaKTet:bcwlcy9) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/make_arm_controller_model.cc#L177)):*
> ```cpp
>   // TODO(eric.cousineau): Is it possible to iterate directly through welded
>   // subgraphs?
>   for (const BodyIndex& sim_body_index : sim_arm_body_indices) {
> ```

BTW Just an artifact of reading the changes due to this PR, but when I read this, I felt myself wondering, "what about things welded to the gripper?". Certainly, the gripper bodies are included in the set of "accounted for" bodies.

That said, I recognize that when handling the gripper, it also handles bodies that are welded to the gripper. Might be worth putting a note here reminding the reader of the code that bodies welded to the gripper were lumped in with the gripper.
___
*[`manipulation/util/make_arm_controller_model.cc` line 177 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne2i-I3BMr73g2-sCU:-Nne2i-I3BMr73g2-sCV:b-klyssa) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/make_arm_controller_model.cc#L177)):*
> ```cpp
>   // TODO(eric.cousineau): Is it possible to iterate directly through welded
>   // subgraphs?
>   for (const BodyIndex& sim_body_index : sim_arm_body_indices) {
> ```

nit: This new behavior should be included in the documentation of the function.

The documentation also fails to mention that bodies welded to the gripper will be lumped in. The `gripper_info` references the gripper model instance index and the language persistently refers to "the gripper". It is not obvious at all that "the gripper" includes any non-arm body that is welded to any gripper body.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 94 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne3jUy9XGI-0KN03CT:-Nne3jUy9XGI-0KN03CU:b-m3ztqk) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/test/make_arm_controller_model_test.cc#L94)):*
nit: This line doesn't feel precise (and possibly incorrect). Could we rephrase this to more accurately reflect when this function can be correctly used?

> This expects that the simulation plant consists only of the arm and (optional) gripper.

As I read this function, it really doesn't have any such restriction. It computes the mass of each plant in total, and the composite inertia of the full plant. So, as long as the plant's "correspond", this function is perfectly happy.

I can see that it might be the case that since the arm controller plant only has arm and gripper (and welded lumped masses as part of those model instances), if the sim plant has any masses that aren't related to any of those bodies, then, it probably won't match the controller plant. So, perhaps the rephrasing might be:

> This test blindly operates on the full simulation and control plants. Therefore, the simulation plant's mass should be wholly contained in bodies that belong to either the arm or (optional) gripper model instances, or be welded to a body in those instances.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 310 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne3HTlEGr0ZLKqgr0J:-Nne3HTlEGr0ZLKqgr0K:b-laie9o) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/test/make_arm_controller_model_test.cc#L310)):*
> ```cpp
>   const std::vector<ModelInstanceInfo> models_from_directives =
>       multibody::parsing::ProcessModelDirectives(directives, &parser);
>   sim_plant_->mutable_gravity_field().set_gravity_vector(
> ```

nit: Changing the gravity seems to be copypasta. It doesn't affect the test outcome so isn't necessary. Or the test doesn't do what it thinks it does. Or there's an arcane purpose that deserves a note.

**Review by SeanCurtis-TRI — COMMENTED** [review]
+(release notes: fix) 

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-:-NneAzPZ-xhjgFvQJAdq:b-chsgxs)* status: 5 unresolved discussions

**Review by EricCousineau-TRI — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)* status: <span>:shipit:</span> complete! all discussions resolved, LGTM from assignees SeanCurtis-TRI(platform),zachfang

___
*[`manipulation/util/make_arm_controller_model.cc` line 87 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nndz9En6bg01-57wN5x:-Nner-o-DN8K2-V77eG-:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/make_arm_controller_model.cc#L87)):*
<details><summary><i>Previously, SeanCurtis-TRI (Sean Curtis) wrote…</i></summary><blockquote>

BTW Perhaps too aggressive a use of `auto` here? Particularly as it's not just a temporary used to instantiate `sim_bodies_accounted_for`, but is used in your new for loop?
</blockquote></details>

Done.
___
*[`manipulation/util/make_arm_controller_model.cc` line 177 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne1tyY69VK4GyaKTes:-Nner0wI3LOu9nKYdRVR:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/make_arm_controller_model.cc#L177)):*
<details><summary><i>Previously, SeanCurtis-TRI (Sean Curtis) wrote…</i></summary><blockquote>

BTW Just an artifact of reading the changes due to this PR, but when I read this, I felt myself wondering, "what about things welded to the gripper?". Certainly, the gripper bodies are included in the set of "accounted for" bodies.

That said, I recognize that when handling the gripper, it also handles bodies that are welded to the gripper. Might be worth putting a note here reminding the reader of the code that bodies welded to the gripper were lumped in with the gripper.
</blockquote></details>

Done
___
*[`manipulation/util/make_arm_controller_model.cc` line 177 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne2i-I3BMr73g2-sCU:-NnertEI7GsNetIWTZsc:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/make_arm_controller_model.cc#L177)):*
<details><summary><i>Previously, SeanCurtis-TRI (Sean Curtis) wrote…</i></summary><blockquote>

nit: This new behavior should be included in the documentation of the function.

The documentation also fails to mention that bodies welded to the gripper will be lumped in. The `gripper_info` references the gripper model instance index and the language persistently refers to "the gripper". It is not obvious at all that "the gripper" includes any non-arm body that is welded to any gripper body.
</blockquote></details>

Done.
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 94 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne3jUy9XGI-0KN03CT:-NnerwwZ6MyehNrNQkRR:b-dtkds3) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/test/make_arm_controller_model_test.cc#L94)):*
<details><summary><i>Previously, SeanCurtis-TRI (Sean Curtis) wrote…</i></summary><blockquote>

nit: This line doesn't feel precise (and possibly incorrect). Could we rephrase this to more accurately reflect when this function can be correctly used?

> This expects that the simulation plant consists only of the arm and (optional) gripper.

As I read this function, it really doesn't have any such restriction. It computes the mass of each plant in total, and the composite inertia of the full plant. So, as long as the plant's "correspond", this function is perfectly happy.

I can see that it might be the case that since the arm controller plant only has arm and gripper (and welded lumped masses as part of those model instances), if the sim plant has any masses that aren't related to any of those bodies, then, it probably won't match the controller plant. So, perhaps the rephrasing might be:

> This test blindly operates on the full simulation and control plants. Therefore, the simulation plant's mass should be wholly contained in bodies that belong to either the arm or (optional) gripper model instances, or be welded to a body in those instances.
</blockquote></details>

Done, thanks!
___
*[`manipulation/util/test/make_arm_controller_model_test.cc` line 310 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nne3HTlEGr0ZLKqgr0J:-Nneryme5Mqdr9-UIGW_:bq2352r) ([raw file](https://github.com/RobotLocomotion/drake/blob/e4d899a25667e1f3b726c99b64fbdb507e9bd712/manipulation/util/test/make_arm_controller_model_test.cc#L310)):*
<details><summary><i>Previously, SeanCurtis-TRI (Sean Curtis) wrote…</i></summary><blockquote>

nit: Changing the gravity seems to be copypasta. It doesn't affect the test outcome so isn't necessary. Or the test doesn't do what it thinks it does. Or there's an arcane purpose that deserves a note.
</blockquote></details>

Done. I had added it to ensure gravity would have an affect (in case things were perfectly balanced), but I've removed it and confirmed this test fails without the functionality update.

**Review by SeanCurtis-TRI — COMMENTED** [review]
> Reviewed 3 of 3 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)* status: 1 unresolved discussion

___
*[`manipulation/util/make_arm_controller_model.h` line 22 at r5](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nnetk8g2QxadLIsDSjT:-Nnetk8h4x-GOvIsXvf4:bjmv1hv) ([raw file](https://github.com/RobotLocomotion/drake/blob/ed83c6b7ac38efc4ff18ec037bdbc58160dd9148/manipulation/util/make_arm_controller_model.h#L22)):*
I'm happy to defer to you but does "resultant inertia" really represent the effect of calling this function does? How about:

> All bodies welded directly to bodies of the arm will be replaced with lumped masses.



_Code quote:_
```objc
the resultant inertia
```

**Review by EricCousineau-TRI — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)* status: 1 unresolved discussion

___
*[`manipulation/util/make_arm_controller_model.h` line 22 at r5](https://reviewable.io/reviews/RobotLocomotion/drake/20733#-Nnetk8g2QxadLIsDSjT:-NneuWeZ6MkeptwLg5tp:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake/blob/ed83c6b7ac38efc4ff18ec037bdbc58160dd9148/manipulation/util/make_arm_controller_model.h#L22)):*
<details><summary><i>Previously, SeanCurtis-TRI (Sean Curtis) wrote…</i></summary><blockquote>

I'm happy to defer to you but does "resultant inertia" really represent the effect of calling this function does? How about:

> All bodies welded directly to bodies of the arm will be replaced with lumped masses.
</blockquote></details>

Done.

**Review by SeanCurtis-TRI — COMMENTED** [review]
> Reviewed 1 of 1 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/20733)* status: <span>:shipit:</span> complete! all discussions resolved, LGTM from assignees SeanCurtis-TRI(platform),zachfang
